resource-timing/TAO-crossorigin-port.sub.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Resource Timing TAO - "null" and opaque origin</title>
+<link rel="author" title="Google" href="http://www.google.com/" />
+<link rel="help" href="https://www.w3.org/TR/resource-timing-2/#timing-allow-origin"/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+
+const t = async_test("Makes sure that the iframe passed the test and had an entry which passed the timing allow check");
+window.addEventListener("message", t.step_func_done(e=>{
+    assert_equals(e.data, "PASS");
+}));
+</script>
+</head>
+<body>
+<h1>Description</h1>
+<p>This test validates that for a cross origin resource, the timing allow check algorithm will succeed when the value of Timing-Allow-Origin is null and the origin is an opaque origin.</p>
+<div id="log"></div>
+<iframe id="frameContext" src="{{location[scheme]}}://{{host]}}:{{ports[http][0]}}/resource-timing/resources/iframe-TAO-crossorigin-port.sub.html"></iframe>
```

Comment:
Is this one meant to be cross-origin? Not clear what part makes it so. Also, I think there is a typo:
```suggestion
<iframe id="frameContext" src="{{location[scheme]}}://{{host}}:{{ports[http][0]}}/resource-timing/resources/iframe-TAO-crossorigin-port.sub.html"></iframe>
```

Reply:
The iframe is on `ports[0]` while the resource it loads is on `ports[1]`, which makes it a cross-origin. (but on the same host)

Thanks for catching the typo